src/tools/r2rdump/R2RMethod.cs

```diff
@@ -37,33 +37,41 @@ struct RuntimeFunction
         /// </summary>
         public int UnwindRVA { get; }
 
-        public RuntimeFunction(int id, int startRva, int endRva, int unwindRva)
+        public R2RMethod Method;
```

Comment:
Is this used? I didn't see it called anywhere. If it is, you should make it a property since its accessibility is public

Reply:
I've made some changes so it prints the method that a runtime function belongs to when it's searched by id